x-pack/plugins/security/public/management/api_keys/api_keys_management_app.tsx

```diff
@@ -46,14 +46,14 @@ export const apiKeysManagementApp = Object.freeze({
         ]);
 
         render(
-          <i18nStart.Context>
-            <APIKeysGridPage
-              navigateToApp={application.navigateToApp}
-              notifications={notifications}
-              docLinks={new DocumentationLinksService(docLinks)}
-              apiKeysAPIClient={new APIKeysAPIClient(http)}
-            />
-          </i18nStart.Context>,
+          <KibanaContextProvider services={{ application, docLinks }}>
```

Comment:
Same as other context provider, it would be safer to pass through the entire `coreStart` object.